build/linux/installer/conf/70-rsyslog-forward-mdsd-ci.conf

```diff
@@ -1,5 +1,10 @@
-$ModLoad omuxsock 
-$OMUxSockSocket /var/run/mdsd-ci/default_syslog.socket 
 template(name="MDSD_RSYSLOG_TraditionalForwardFormat" type="string" string="<%PRI%>%TIMESTAMP% %HOSTNAME% %syslogtag%%msg:::sp-if-no-1st-sp%%msg%")
-$OMUxSockDefaultTemplate MDSD_RSYSLOG_TraditionalForwardFormat
-*.* :omuxsock:
\ No newline at end of file
+
+# Forwarding all events through TCP port
+*.* action(type="omfwd"
+template="MDSD_RSYSLOG_TraditionalForwardFormat"
+queue.type="LinkedList"
+action.resumeRetryCount="-1"
+queue.size="10000"
```

Comment:
is the queue size for rsyslog service buffer the data? is 10k records/events?

Reply:
this is only when connection breaks, so I think 10K is ok for now.